### ISSUE TITLE
Garden trying to evaluate modules while linking source

### ISSUE THREAD

**Issue by failshell** [garden-io/garden#2283]
### Current Behavior

Garden is trying to evaluate modules based on the defined sources in the project while linking sources. That leads to an error.

~~~
☸ ops in garden-as-code/projects/hello-world on  master [?] execution time: 4s
2021-03-01 08:27:48 ❯ g link source shared ~/src/garden/garden-shared
Link source 🔗

✔ garden-enterprise         → Ready
Templated module linting-dockerfile references template helper-generate-hadolint-configuration, which cannot be found. Available templates: helper-wait-for-ingress

See error.log for detailed error message
~~~

### Expected behavior

I should be able to link sources without garden evaluating the defined sources in the project.

### Workaround

Comment module and link, then uncomment the module.

### Your environment

* **OS:** macOS
* **garden version:** 0.12.17

**Comment by eysi09** [comment]
Thanks for the report. I'll look into this.

**Comment by vvagaytsev** [comment]
Closing this as fixed in Bonsai (Garden 0.13). @failshell please let us know if this is still an issue.
